doc/classes/RefCounted.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		Base class for any object that keeps a reference count. [Resource] and many other helper objects inherit this class.
 		Unlike other [Object] types, [RefCounted]s keep an internal reference counter so that they are automatically released when no longer in use, and only then. [RefCounted]s therefore do not need to be freed manually with [method Object.free].
+		Cycles of [RefCounted] instances will [b]not[/b] be automatically freed. For example, if a node holds a reference to instance [code]A[/code], which directly or indirectly holds a reference back to [code]A[/code], [code]A[/code]'s reference count will be 2. Destruction of the node will leave [code]A[/code] dangling with a reference count of 1, and the memory is leaked. To prevent this, one of the references in the cycle can be made weak with [method @GlobalScope.weakref].
```

Comment:
```suggestion
		[RefCounted] instances caught in a cyclic reference will [b]not[/b] be freed automatically. For example, if a node holds a reference to instance [code]A[/code], which directly or indirectly holds a reference back to [code]A[/code], [code]A[/code]'s reference count will be 2. Destruction of the node will leave [code]A[/code] dangling with a reference count of 1, and there will be a memory leak. To prevent this, one of the references in the cycle can be made weak with [method @GlobalScope.weakref].
```

A slight stylistic change. I'm a bit concerned about the "cycles of" intro, I don't think it's very clear what that means. So hopefully this is better.